backend/eventloop/application_event_loop/application_event_runner_deployments.go

```diff
@@ -321,6 +321,19 @@ func (a applicationEventLoopRunner_Action) handleNewGitOpsDeplEvent(ctx context.
 	}
 	a.log.Info("Created new Application in DB: "+application.Application_id, application.GetAsLogKeyValues()...)
 
+	// Create ApplicationOwner row in DB
+	applicationOwner := db.ApplicationOwner{
+		ApplicationOwnerApplicationID: application.Application_id,
+		ApplicationOwnerUserID:        clusterUser.Clusteruser_id,
+	}
+
+	if err := dbQueries.CreateApplicationOwner(ctx, &applicationOwner); err != nil {
```

Comment:
Based on the discussion in standup, it's possible that the application owner row already exists, so the code should check for that and skip the creation in that case.